packages/lab/src/tabs-next/TabNext.css

```diff
@@ -26,7 +26,7 @@
 
 .saltTabNext:hover {
   cursor: var(--salt-navigable-cursor-hover);
-  background: var(--salt-navigable-primary-background-hover);
+  background: var(--salt-navigable-background-hover);
```

Comment:
Why remove the background per variant naming convention - will each of the variants , ( primary, secondary, cta) not need different background hover colours ?

Reply:
there isn't a ..navigable-secondary.. or ..navigable-cta.. to begin with so i think it's design standardising/simplifying token names